server/src/test/java/org/elasticsearch/reservedstate/service/ReservedClusterStateServiceTests.java

```diff
@@ -367,9 +334,8 @@ public void onFailure(Exception e) {}
         );
 
         var trialRunResult = controller.trialRun("namespace_one", state, chunk, new LinkedHashSet<>(orderedHandlers));
-        assertEquals(0, trialRunResult.nonStateTransforms().size());
-        assertEquals(1, trialRunResult.errors().size());
-        assertTrue(trialRunResult.errors().get(0).contains("Error processing one state change:"));
+        assertThat(trialRunResult.nonStateTransforms(), empty());
+        assertThat(trialRunResult.errors(), contains(containsString("Error processing one state change:")));
```

Comment:
Does this lose the assertion that `trialRunResult.errors()` contains just one entry?